### PR TITLE
docs - add optional procedure to cfg pxf host/port

### PR DESCRIPTION
**Pull request by lisakowen**
in this PR:
- create new page "Configuring the PXF Agent Host and Port (Optional)", nest under the "Configuring PXF" left subnav entry
- this task is not a mainline configuration task; adding only to the subnav, not referenced on the "Configuring PXF" page

doc review site link:  http://docs-lisa-pxfhostport.cfapps.io/6-0/pxf/cfghostport.html

### CODE REVIEW

File: gpdb-doc/markdown/pxf/cfghostport.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: Configuring the PXF Agent Host and Port (Optional)
+---
+
+By default, a PXF agent started on a segment host listens on port number `5888` on `localhost`. You can configure PXF to start on a different port number, or use a different hostname or IP address to identify the host. If your Greenplum Database cluster requires this custom configuration, you will set one or both of the environment variables identified below:
```
Comment (dyozie) [inline gpdb-doc/markdown/pxf/cfghostport.html.md.erb]: remove "to identify the host" here (or change the wording).  You're not affecting the host here, just the PXF agent.  I'm guessing this is just for multihomed host machines.

also:

If your Greenplum Database cluster requires this custom configuration, you will -> To change the default configuration,

File: gpdb-doc/markdown/pxf/cfghostport.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+|   Environment Variable  |  Description      |
+--------------------------+--------------------
+|   PXF_HOST  | The name of the host. The default host name is `localhost`.  |
```
Comment (dyozie) [inline gpdb-doc/markdown/pxf/cfghostport.html.md.erb]: host -> host or IP Address to use for PXF.

(or something similar).

File: gpdb-doc/markdown/pxf/cfghostport.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+|   PXF_PORT  | The port number on which the PXF agent listens for requests on the host. The default port number is `5888`.  |
+
+You set the environment variables in the `gpadmin` user's `.bashrc` shell login file on each segment host.
```
Comment (dyozie) [inline gpdb-doc/markdown/pxf/cfghostport.html.md.erb]: You set -> Set

File: gpdb-doc/markdown/pxf/cfghostport.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+<div class="note">You must restart both Greenplum Database and PXF when you configure the agent host and/or port in this manner. Consider performing this configuration during a scheduled down time.</div>
+
+## <a id="proc"></a>Procedure
```
Comment (dyozie) [inline gpdb-doc/markdown/pxf/cfghostport.html.md.erb]: Wondering if the rest of this (the actual Procedure) is even necessary?  I think we should expect readers to make the adjustment just given the info above.

Comment (lisakowen) [reply gpdb-doc/markdown/pxf/cfghostport.html.md.erb]: good question.  they could choose to change the pxf port on only one segment host.  or they configure different port numbers on all hosts.  the procedure seemed a good way to over the bases.

**Comment by lisakowen** [comment]
@frankgh - did not include master in these config instructions - wasn't sure what the admin would set on master since:
- the pxf agent does not currently run on master, and
- the segment hosts could all have different hosts and port numbers

let me know if i am missing something.

**Comment by frankgh** [comment]
@lisakowen I think minimal documentation works for this feature, so we can skip documenting master in this instructions

**Pull request merged**